### PR TITLE
Base login

### PR DESCRIPTION
**Pull request by VeeAndrade**
#### What's this PR do?
This PR goes through iteration 1 and 2, does not display all data yet but I wanted to jump on to another branch before I started that. This branch follows TDD workflow, all tests were written bore my code, and all tests are currently passing.
#### Where should the reviewer start?
Reading the commit messages will give you an idea of the workflow.
#### How should this be manually tested?
If you wanted to run the test manually on your machine, you could pull this pr down and run it.
@kaylaewood

### CODE REVIEW

**Comment by kaylaewood** [comment]
@VeeAndrade ! This looks SO good! I like the way you've structured your classes. Only two classes and neither feels like they are too big. Also, great job writing tests for each method!

A couple of things I would add:
- Add testing for each of the properties in both classes. In our spec, it says to run tests for all "class default properties." So, you want to make sure things like id and bookings are actually populating the way you think they are.
- Remove the sample-test file. That was a smart idea to make sure everything was synced up, but now that you have tests passing in the terminal, you don't need that file any more.

File: src/customer.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Customer {
+  constructor(id, bookings, rooms){
+    this.id = id;
+    this.bookings = bookings;
+    this.rooms = rooms;
```
Comment (kaylaewood) [inline src/customer.js]: Make sure you test these properties!

File: src/manager.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
+class Manager {
+  constructor(bookings, rooms) {
+    this.bookings = bookings;
+    this.rooms = rooms;
```
Comment (kaylaewood) [inline src/manager.js]: Test these too!

File: src/customer.js (hunk 1 above)
Comment (VeeAndrade) [reply src/customer.js]: Totally forgot to go back and do that. Thanks for the catch.

**Pull request merged**